src/createCycleMiddleware.js

```diff
@@ -1,4 +1,5 @@
 import xs from 'xstream'
+import {adapt} from '@cycle/run/lib/adapt'
```

Comment:
Good point. Will it work though? If the user is using rxjs-run, then the sinks will be RxJS streams, hence `xs.merge(rxjsOsbervable)` will break. How do we make it work?